api/tools/src/main/java/org/pmiops/workbench/tools/FixDesynchronizedBillingProjectOwners.java

```diff
@@ -85,7 +86,7 @@ private static void dryLog(boolean dryRun, String msg) {
 
   private static void clean(
       WorkspacesApi workspacesApi,
-      BillingApi billingApi,
+      BillingV2Api billingV2Api,
```

Comment:
Why V2 only here?  This looks like it should be deciding based on feature flag.

Reply:
See https://docs.google.com/document/d/1RXWgiFvIEku0TYDsvRhKCkbQ0CNWZcM3mu9r1ZP0Rds/edit#heading=h.28rj3m1by7ms
`Get /api/billing/{projectId}/members` and `Put & delete: /api/billing/{projectId}/{role}/{email}:`
are the same for v1 and v2, and ready to migrate anytime. To keep simply, we will just switch to v2 directly. Does that SG?

Reply:
hm.  I'm not really familiar enough with this tool to say